docs/reference/API/element.md

```diff
@@ -0,0 +1,295 @@
+
+## Methods and Properties
+
+| Property | Description                             |
+|----------|-----------------------------------------|
+| `element`   | Returns the element with the given ID. |
+| `id`        | Returns the element's ID.              |
+| `value`    | Returns the element's value.           |
+| `innerHtml` | Returns the element's inner HTML.      |
+
+
+
+| Method               | Description                            |
+|----------------------|--------------------------------------------------------------|
+| `write`                | Writes `value` to element and handles various mime types. `append` defaults to `False`, if set to true, it will create a child element.          |
+| `clear`                | Clears the element's value or content. |
+| `select`               | Select element from `query`.    |
+| `clone`                | Clones the with `new_id` if provided and `to` element if provided.                   |
+| `remove_class`         | Removes one or more class name from the element.    |
+| `add_class`            | Adds one  or more class name to the element.    |
+
+## Element.element
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+The `element` property returns the DOM element with the given ID.
+
+```html
+    from pyscript import Element
+
+    my_div = Element('my-div')
+    print(my_div.element)
+```
+
+## Element.id
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's ID.
+
+```html
+
+<div id="my-div"></div>
+
+<py-script>
+    from pyscript import Element
+
+    my_div = Element('my-div')
+    print(my_div.id) # prints 'my-div'
+</py-script>
+```
+
+## Element.value
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's value.
+
+```html
+<input id="my-input" value="hello world"></input>
+
+<py-script>
+    from pyscript import Element
+
+    my_input = Element('my-input')
+    print(my_input.value) # prints 'hello world'
+</py-script>
+```
+
+## Element.innerHtml
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's inner HTML.
+
+```html
+<div id="my-innerHtml">
+    <b>hello world</b>
+</div>
+
+<py-script>
+    from pyscript import Element
+
+    my_innerHtml = Element('my-innerHtml')
+    print(my_innerHtml.innerHtml) # prints <b> hello world </b>
+</py-script>
+```
+
+## Element.write
+
+| Parameter   | Default | Type                        |
+|-------------|---------|-----------------------------|
+| `value`     |         |  `str` or `__mime_type__`   |
+| `append`    | False   | `bool`                      |
+
+Writes `value` to element and handles various mime types. This method also contains a `append` parameter, which defaults to `False`.
```

Comment:
We're deprecating `Element.write`
https://github.com/pyscript/pyscript/blob/5aa9135a344822bf2e250a0bfbc78f58bdab74bb/pyscriptjs/src/python/pyscript.py#L134
We should have it in the docs :)

Reply:
Just to confirm, he thing that we are deprecating is `pyscript.write` not `element.write` right? 🤔

Reply:
So that's an invalid comment, thanks for noticing.

Reply:
ah gotcha, i was just wondering if I missed something on discord 😄